setup.py

```diff
@@ -341,11 +341,19 @@ def getExtensionModules(nupicCoreReleaseDir, platform, bitness):
   commonLinkFlags = [
     "-m" + bitness,
     "-fPIC",
-    "-L" + pythonPrefix + "/lib"]
+    "-L" + pythonPrefix + "/lib",
+    "-L" + nupicCoreReleaseDir + "/lib",
```

Comment:
The three lines below specify libraries to link to and this line makes sure it looks in the right place.